ml-proto/test/float_exprs.wast

```diff
@@ -500,44 +500,54 @@
 ;; Test that 1.0/x isn't approximated.
 
 (module
-  (func $f32.no_approximate_reciprocal (param $x f32) (param $y f32) (result f32)
+  (func $f32.no_approximate_reciprocal (param $x f32) (result f32)
     (f32.div (f32.const 1.0) (get_local $x)))
   (export "f32.no_approximate_reciprocal" $f32.no_approximate_reciprocal)
 )
 
-(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const -0x1.2900b6p-10) (f32.const 0x1.d427e8p+56)) (f32.const -0x1.b950d4p+9))
-(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const 0x1.e7212p+127) (f32.const -0x1.55832ap+44)) (f32.const 0x1.0d11f8p-128))
-(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const -0x1.42a466p-93) (f32.const 0x1.7b62d8p+36)) (f32.const -0x1.963ee6p+92))
-(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const 0x1.5d0c32p+76) (f32.const 0x1.d14dccp-74)) (f32.const 0x1.778362p-77))
-(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const -0x1.601de2p-82) (f32.const -0x1.3c7abap+42)) (f32.const -0x1.743d7ep+81))
+(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const -0x1.2900b6p-10)) (f32.const -0x1.b950d4p+9))
+(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const 0x1.e7212p+127)) (f32.const 0x1.0d11f8p-128))
+(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const -0x1.42a466p-93)) (f32.const -0x1.963ee6p+92))
+(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const 0x1.5d0c32p+76)) (f32.const 0x1.778362p-77))
+(assert_return (invoke "f32.no_approximate_reciprocal" (f32.const -0x1.601de2p-82)) (f32.const -0x1.743d7ep+81))
 
-;; Test that 1.0/sqrt(x) isn't approximated.
+;; Test that 1.0/sqrt(x) isn't approximated or fused.
 
 (module
-  (func $f32.no_approximate_reciprocal_sqrt (param $x f32) (param $y f32) (result f32)
+  (func $f32.no_approximate_reciprocal_sqrt (param $x f32) (result f32)
     (f32.div (f32.const 1.0) (f32.sqrt (get_local $x))))
   (export "f32.no_approximate_reciprocal_sqrt" $f32.no_approximate_reciprocal_sqrt)
+
+  (func $f64.no_fuse_reciprocal_sqrt (param $x f64) (result f64)
+    (f64.div (f64.const 1.0) (f64.sqrt (get_local $x))))
+  (export "f64.no_fuse_reciprocal_sqrt" $f64.no_fuse_reciprocal_sqrt)
```

Comment:
FWIW, you can now merge the export into the function def, avoiding all the repetition.